roles/common_charts/files/descheduler/cronjob.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: batch/v1beta1
+apiVersion: batch/v1
```

Comment:
nice, good catch.  This should be fine as that new API was rolled out long enough ago